ccsljava_engine/plugins/org.eclipse.gemoc.execution.concurrent.ccsljavaengine/src/org/eclipse/gemoc/execution/concurrent/ccsljavaengine/dse/SynchroneExecution.java

```diff
@@ -41,21 +41,16 @@ public void run() {
 	}
 
 	/**
-	 * Calls the {@link EventExecutor} for the given
-	 * {@link EngineEventOccurence}.
+	 * Calls the {@link EventExecutor} for the given {@link EngineEventOccurence}.
 	 * 
-	 * @param mse
-	 *            the {@link EngineEventOccurence} to execute
+	 * @param mse the {@link EngineEventOccurence} to execute
 	 * @return the {@link FeedbackData} if any, <code>null</code> other wise
+	 * @throws CodeExecutionException
 	 */
-	private Object callExecutor() {
+	private Object callExecutor() throws CodeExecutionException {
 		Object res = null;
 
-		try {
-			res = getExecutionContext().getExecutionPlatform().getCodeExecutor().execute(getSmallStep().getMseoccurrence());
-		} catch (CodeExecutionException e) {
```

Comment:
FYI this part here is to make sure to crash the execution engine when the code executor does not succeed to call an operation. While this is a bit out the scope of this PR, I believe this is better.